examples/apps/collaborative-textarea/jest.config.cjs

```diff
@@ -4,11 +4,16 @@
  */
 
 // Get the test port from the global map and set it in env for this test
-const testTools = require("@fluidframework/test-tools");
+let mappedPort = 9000; // Defaul port test-tools would provide if no mapping exists
+// Only import test-tools if POLICY_CHECK is not set.
+// This allows us to run policy checks on the jest config files without having to build the repo.
+if (process.env.POLICY_CHECK === undefined) {
+	const testTools = require("@fluid-private/test-tools");
+	mappedPort = testTools.getTestPort(name);
+}
```

Comment:
All jest config files were updated to skip importing test-tools when the new POLICY_CHECK env variable is set, so that we can run policy check on them without having to build the client release group. The `npm-package-json-script-jest-config` handler imports/requires the jest config files, which in turn causes them to try to require from test-tools, but since this PR makes test-tools part of the client release group, if the release group hasn't been built then trying to require from that package fails.